recipes/meraculous/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+    name: meraculous
+    version: "2.2.4"
+
+source:
+    fn: Meraculous-v2.2.4.tar.gz    
+    url: https://sourceforge.net/projects/meraculous20/files/Meraculous-v2.2.4.tar.gz
+    md5: 349feb6cb178643a46e4b092c87bad3a
+
+build:
+    skip: True  # [osx]
+    number: 0
+
+requirements:
+    build:
+    - cmake >=2.8
+    - gcc ==4.8.5 # [linux]
```

Comment:
you don't need to pin gcc here

Reply:
This is now removed by my second commit.